.github/workflows/build.yml

```diff
@@ -7,14 +7,20 @@ on:
     branches: [ main ]
 
 jobs:
-  build:
+  check:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v3
         with:
           fetch-depth: 20
+      - run: make install-check-tools
+      - run: make check-ltag
+      - run: make check-dco
+      - run: make check-lint
+  test:
```

Comment:
This runs in parallel. Does it maybe make more sense to only trigger this if `check` passes?

Reply:
I think it's a better experience for contributors to be able to see all of the automated blockers up front. If you have both a missing license header and a broken test, it would be pretty annoying to have to discover that serially.

Reply:
Fair enough.